config/src/main/java/org/mobilitydata/gtfsvalidator/config/DefaultConfig.java

```diff
@@ -78,6 +81,15 @@ public DefaultConfig(final Logger logger) {
             e.printStackTrace();
         }
         specRepo = new InMemoryGtfsSpecRepository(gtfsSpecProtobufString, gtfsSchemaAsString);
+
+        this.usecaseByFilename = new HashMap<>();
+        final List<FileSpecificUsecase> routeRelatedUsecaseCollection = new ArrayList<>();
+        routeRelatedUsecaseCollection.add(validateRouteShortNameLength());
```

Comment:
As discussed, this will not scale very well. One possible solution is to have specific use cases to retrieve the associated checks for a file